sdk/modelsrepository/azure-iot-modelsrepository/src/samples/README.md

```diff
@@ -18,7 +18,7 @@ The samples project demonstrates the following:
 ```java
 // When no URI is provided for instantiation, the Azure IoT Models Repository global endpoint
 // https://devicemodels.azure.com/ is used and the model dependency resolution
-// configuration is set to TRY_FROM_EXPANDED.
+// configuration is set to DISABLED.
```

Comment:
Model dependency resolution should be enabled by default.

Reply:
Yeah the actual logic has this set to enabled, the documentation needs to be fixed.